tex/chapters/08-Conclusion.tex

```diff
@@ -0,0 +1,51 @@
+Ultimately both of these goals were achieved and some preliminary analysis of the Axelrod-Python was completed.
+In particular, an \textbf{original} results is presented in Section \ref{sec:fsm_proof} allowing the work completed in previous literature to be extended further.
+
+
+\section{Overview}
+
+As described in Chapter \ref{cha:introduction}, the Prisoner's Dilemma is a very popular model in game theory and there have been many papers written about the subject.
+The game has been applied to many different research areas is often used to model systems in biology \cite{Sigmund1999}, sociology \cite{Franken2005}, psychology \cite{Ishibuchi2005}, and economics \cite{Chong2005}.
+In Chapter \ref{cha:literature} an overview of literature related to Prisoner's Dilemma was presented, with particular emphasis on the IPD and several computer tournaments that have be held.
+This was followed by a description of several common strategies that have been used throughout the report.
+Also, an outline of literature relating to Finite State Machines and how they are relevant within IPD is given.
+This is follow by a description of how Axelrod's work is currently being reproduced by an open-source community.
+Finally, an introduction to Fingerprinting (a technique used for identifying similar strategies) is given.
+
+In Chapter \ref{cha:theory} many definitions were presented culminating in a rigorous definition for a Fingerprint.
+It was then shown that any strategy can be represented as a Finite State Machine, allowing the work of Ashlock to be extended to include any strategy to be used as a probe.
+Finally, an example of how to construct an Analytical Fingerprint was provided.
+
+In Chapter \ref{cha:implementation}, the process of implementing Fingerprint in Axelrod-Python is explained, including several algorithms equivalent to definitions from Chapter \ref{cha:theory}.
+Comparisons with work produced in previous literature are made to ensure that results produced are correct.
+
+In Chapter \ref{cha:results} several fingerprints were examined in detail.
+This included discussions of individual fingerprints, a comparison of fingerprints for a strategy with a varying parameter, and also the importance of probe selection.
+
+In Chapter \ref{cha:machinelearning} a new approach to identifying similar strategies using Machine Learning was given.
+The process of training and assessing the model was described in detail, followed by some preliminary results of the models performance.
+Finally, in Chapter \ref{cha:applying-model}, the model was used to identify equivalent strategies.
+First, a level of the model's sensitivity was obtained using a variety of MemoryOne strategies from Axelrod-Python.
+This was followed by the model being applied to all strategies within Axelrod-Python; producing network graphs of strategies the model considered identical.
```

Comment:
and then? This (and that chapter needs a bit more). The point of that chapter is to see how good/useful the model is, what do the experiments say about that? 

The main point of the networks is that the ones it's picked up as similar make sense so to speak.